docs/designs/link-output.md

```diff
@@ -0,0 +1,97 @@
+# Link And Bookmarks Output
+
+Docfx generate `links` and `bookmarks` information of each files into one single file `links.json` for future data utilization of downstream service of docfx.
```

Comment:
do we need to a config flag to control the generation of this file (as well as other files like .xrefmap.json) since it is only useful for English master/live commit build

Reply:
Even we have this config, we still need handle non-English cases, and we don't want different config in different branches.